packages/ui/src/ui/components/UserLink/UserLink.tsx

```diff
@@ -18,14 +18,19 @@ interface Props {
     children?: React.ReactElement;
 }
 
-export function UserCard({userName, className, noLink, internal, children}: Props) {
+export function DefaultUserCard(props: UserCardProps) {
+    const {userName, className, noLink, internal, children} = props;
     const url = internal
         ? `/${YT.cluster}/users?filter=${userName}`
         : UIFactory.makeUrlForUserName({login: userName, cluster: YT.cluster});
 
     return <UserName {...{className, userName, url: noLink ? undefined : url, children}} />;
 }
 
+export function UserCard(props: UserCardProps) {
```

Comment:
It looks like the component is never used. What is the reasone to add it?

Reply:
This is an [old](https://github.com/ytsaurus/ytsaurus-ui/pull/479/files#diff-398b6bf172e159d1d5b19c38b72ab860de0139fd132496b4b970bc5766202dc5L21) `UserCard` component that was used in the application. I pulled the code from it into `DefaultUserCard`, and inside `UserCard` I left the use of the plugin from UIFactory.